src/main/scala/edu/ie3/simona/scheduler/TimeAdvancer.scala

```diff
@@ -53,21 +45,18 @@ object TimeAdvancer {
     *   last tick of the simulation
     */
   def apply(
-      simulation: org.apache.pekko.actor.ActorRef,
+      simulation: ActorRef[SimonaSim.SimulationEnded.type],
```

Comment:
Yes, because `SimulationEnded` is a `case object`. Just using `SimulationEnded` would refer to the singular instance, but you want to refer to the type here.

So it works a bit like this:
| Entity | Type | Instance |
|-|-|-|
| class A | A | new A() |
| object B | B.type | B |